src/routes/api.ts

```diff
@@ -24,14 +24,16 @@ import {
   SubracesHandler,
   TraitsHandler,
   WeaponPropertiesHandler,
-} from './api/index.js';
+} from './api/2014/index.js';
+import V2014Handler from './api/2014.js';
 
 import express from 'express';
 import { index } from '../controllers/apiController.js';
 
 const router = express.Router();
 
 router.get('/', index);
+router.use('/2014', V2014Handler);
```

Comment:
Are we sure we wanna do this or have 2014 be the default and have people opt into 2024? I think most users are using the 2014 version and this would be a big breaking change.

Reply:
Both original and 2014 routes will exist after this. This is not a breaking change.